### DISCUSSION TITLE
This mod in China NetEase version of Minecraft

### DISCUSSION THREAD

**Discussion by nanshengcn** [vectorwing/FarmersDelight#946] [General]
In the Chinese version of Minecraft that Netease represents, a studio created a mod similar to this one, claiming that the mod was for the fun of the farmers, is this mod distributed with your permission?
Here's their mod launch trailer:https://b23.tv/IF6B2vJ

**Comment by vectorwing** [comment]
I have not given permission for my mod to be distributed in those platforms. Sadly, due to the nature of China's copyright laws, as well as the platforms where these are hosted, there isn't much I can do about it.

If you're able, feel free to report these posts in the respective websites.

**Reply by nanshengcn** [reply]
Unfortunately, they made a fake Github Q&A with someone who wasn't part of the mod you made at all, who stated that you don't have permission to manage the mod, and the studio self-righteously claimed that the mod it made was authorized by you. There are also people in the video who call this mod studio the godfather, which is ridiculous.
![Screenshot_20240730_161334_tv danmaku bili](https://github.com/user-attachments/assets/ccaf56c4-4319-4ce2-874b-dfa5fb8e936b)
But don't worry at the moment, I have submitted a report to NetEase, and soon this infringing mod will be taken down.

**Reply by vectorwing** [reply]
RaymondBlaze didn't really say anything wrong here. He's correct in that I hold no copyright over *add-ons* aside from their usage of my code and assets, and as such, their own copyrights must be respected.

The MIT License technically allows commercial usage of FD, so long as they clearly state my copyright and authorship within their uses. If they didn't do that (a.k.a they stole the project and passed it as their own creation), then it's infringement.

**Reply by vectorwing** [reply]
Thanks for the support regarding this, by the way. There should be other discussion threads about this problem on this board as well. 👍

**Comment by vectorwing** [comment]
I just realized that this is referring to a **Bedrock Edition** port of FD, and not a commercial resale of the Forge mod! 🤦

The author of this add-on actually coded the port on their own, and therefore it's totally fine. In fact, I only now realized that they asked for permission in this discussion board; I never replied due to my extended break.

I apologize for the confusion. @nanshengcn, please do not report this add-on, if possible.

**Reply by nanshengcn** [reply]
Actually, this is not true. They copy your mod and will later indicate the original author and the MIT license. To this day, I opened NetEase's creator platform and found that this mod still does not indicate the original author and MIT license, and claims that it has been authorized. In fact, there are many mods on the NetEase creator platform that are similar to the mods made by your studio, and some even charge a fee. Also, this mod is intended to copy your mod affiliate pack and charge for it in the future.

**Reply by nanshengcn** [reply]
What they noted when sending you a Github discussion was "Some players think I used your code", which is actually not true. Not only does this mod have many bugs, but then it still doesn't indicate that your mod developer is licensed with MIT, which is infringing.

**Reply by nanshengcn** [reply]
They only write two lines about the developer of your mod and the MIT license.
![Uploading Screenshot_20240730_201227_com.netease.mc.huawei.jpg…]()

**Reply by nanshengcn** [reply]
![Screenshot_20240730_201227_com netease mc huawei](https://github.com/user-attachments/assets/042501c4-4ba2-48cd-b73e-91daafa6adc8)
